docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -712,10 +718,6 @@ public void pullImage(String imageName) {
 
             PullImageCmd pullImageCmd = this.dockerClient.pullImageCmd(image.getName());
 
-            if (this.cubeConfiguration.getDockerRegistry() != null) {
-                pullImageCmd.withRegistry(this.cubeConfiguration.getDockerRegistry());
```

Comment:
Ok, so docker client executor and cubeConfiguration is per image? What if I have 1 image available in my private repo and other from the docker hub? Are we able to handle it?